template/en/default/attachment/create.html.tmpl

```diff
@@ -84,7 +84,7 @@ TUI_hide_default('attachment_text_field');
         <td>
           <em>If you want to assign this [% terms.bug %] to yourself,
               check the box below.</em><br>
```

Comment:
I guess this should say "If you want to assign this bug to yourself, uncheck the box below."
otherwise LGTM.